ci-operator/step-registry/ipi/install/hosted-loki-staging/ipi-install-hosted-loki-staging-commands.sh

```diff
@@ -392,6 +394,7 @@ spec:
   - configMap
   - hostPath
 EOF
+fi
```

Comment:
i don't think you need the rbac right below this either, since it is granting permission to use the psp